stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -22,9 +22,9 @@ function Bidiagonal{T}(dv::AbstractVector, ev::AbstractVector, uplo::Union{Symbo
                convert(AbstractVector{T}, ev)::AbstractVector{T},
                uplo)
 end
-function Bidiagonal{T,V}(A::Bidiagonal) where {T,V<:AbstractVector{T}}
-    Bidiagonal{T,V}(A.dv, A.ev, A.uplo)
-end
+# function Bidiagonal{T,V}(A::Bidiagonal) where {T,V<:AbstractVector{T}}
+#     Bidiagonal{T,V}(A.dv, A.ev, A.uplo)
+# end
```

Comment:
The corresponding method for `A::AbstractMatrix` doesn't even exist! The same applies to `[Sym]Tridiagonal`. That's not generic programming, right? If you specify `T` and `V`, you already know that you're working with a, for instance, `Bidiagonal` here.

Reply:
Ok, I realized these are actually needed for the `convert` methods, so perhaps rather internal.